example_config.toml

```diff
@@ -0,0 +1,20 @@
+# All fields except bot.token are optional, defaults shown below
+
+[behavior]
+max_keywords = 100
+patience_seconds = 120
+
+[bot]
+#token = "..."
+private = true
```

Comment:
I think it would make more sense for the `[bot]` section to be at the top of the example config. Also, it seems like the default for `private` is `false` (which is good) but this implies it's `true`.